internal/tiltfile/local_resource.go

```diff
@@ -45,6 +47,7 @@ func (s *tiltfileState) localResource(thread *starlark.Thread, fn *starlark.Buil
 		"serve_cmd?", &serveCmdVal,
 		"cmd_bat?", &updateCmdBatVal,
 		"serve_cmd_bat?", &serveCmdBatVal,
+		"force_parallel?", &forceParallel,
```

Comment:
`allow_parallel` would feel a bit clearer to me, but not a blocker

Reply:
done!